test_fms/fms2_io/test_global_att.F90

```diff
@@ -31,14 +33,21 @@ program test_global_att
 integer(kind=i4_kind), dimension(2)    :: buf_i4_kind_1d !< i4_kind 1D buffer
 integer(kind=i8_kind)   :: buf_i8_kind !< i8_kind buffer
 integer(kind=i8_kind), dimension(2)    :: buf_i8_kind_1d !< i8_kind 1D buffer
+character (len = 120), dimension(3) :: my_format !> Array of formats to try.
 
 character(len=20) :: buf_str !< character buffer
+integer :: i !< For Do loop
```

Comment:
!< should be !>

Reply:
I'm confused here, our style guide shows that the original code is correct "!<"

Reply:
I was going off of the Fortran comment example in the [doxygen manual ](http://cs.swan.ac.uk/~csoliver/ok-sat-library/internet_html/doc/doc/Doxygen/1.7.6.1/html/docblocks.html), but maybe the use of !> or !< doesn't matter for starting lines. I recall having problems rendering the html pages when using !< for subroutine comments, and !> for variables, but maybe this is no longer the case (or I was doing something wrong).

> For Fortran "!>" or "!<" starts a comment and "!!" or "!>" can be used to continuate a one line > comment into a multi-line comment.
>
> Here is an example of a documented Fortran subroutine:
```
  !> Build the restriction matrix for the aggregation 
  !! method.
  !! @param aggr information about the aggregates
  !! @todo Handle special case
  subroutine IntRestBuild(A,aggr,Restrict,A_ghost)
    implicit none
    Type(SpMtx), intent(in) :: A !< our fine level matrix
    Type(Aggrs), intent(in) :: aggr
    Type(SpMtx), intent(out) :: Restrict !< Our restriction matrix

```

Reply:
I believe that is still the case, doxygen's documentation makes them seem interchangeable but when it's actually parsed it tends to miss `!>` if it's at the end of a line so it's best to have `!<` after variables